From 15329c5a0d3a5d9a8eddf0bed38c3b62c7fc7965 Mon Sep 17 00:00:00 2001
From: Sebastian Wick <sebastian@sebastianwick.net>
Date: Thu, 31 Oct 2019 15:17:26 +0100
Subject: [PATCH] XF86keysym: Add XF86XK_FullScreen

Add XF86XK_FullScreen keysym, to be used as mapping for evdev's
KEY_FULL_SCREEN.

Chromebooks have a special media key for toggling full screen mode.

Signed-off-by: Sebastian Wick <sebastian@sebastianwick.net>
---
 include/X11/XF86keysym.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/X11/XF86keysym.h b/include/X11/XF86keysym.h
index dd287e2..dab1220 100644
--- a/include/X11/XF86keysym.h
+++ b/include/X11/XF86keysym.h
@@ -207,6 +207,8 @@
 
 #define XF86XK_RotationLockToggle 0x1008FFB7 /* Toggle screen rotation lock on/off */
 
+#define XF86XK_FullScreen	0x1008FFB8   /* Toggle fullscreen */
+
 /* Keys for special action keys (hot keys) */
 /* Virtual terminals on some operating systems */
 #define XF86XK_Switch_VT_1	0x1008FE01
-- 
2.26.2

